### DISCUSSION TITLE
[error] error: [Dagger/MissingBinding] com.yangbong.data.remote.data_source.RemoteSelectCharacterDataSource cannot be provided without an @Provides-annotated method.

### DISCUSSION THREAD

**Discussion by yoonhyeokCho** [TEAM-DAME/Dame-Android#73] [2. Q&A]
![image](https://user-images.githubusercontent.com/90389363/202455836-369198b7-84a7-42e0-aba3-328d46e47880.png)
![image](https://user-images.githubusercontent.com/90389363/202456237-cae27366-b699-476f-90d2-d156bba149b7.png)

# Error Message

/Users/yoonhyeok/AndroidStudioProjects/DameDame-Android/app/build/generated/source/kapt/debug/com/yangbong/damedame/DameDameApplication_HiltComponents.java:161: error: [Dagger/MissingBinding] com.yangbong.data.remote.data_source.RemoteSelectCharacterDataSource cannot be provided without an @Provides-annotated method.
  public abstract static class SingletonC implements DameDameApplication_GeneratedInjector,
                         ^
      com.yangbong.data.remote.data_source.RemoteSelectCharacterDataSource is injected at
          com.yangbong.data.repository.SelectCharacterRepositoryImpl(selectCharacterDataSource, …)
      com.yangbong.data.repository.SelectCharacterRepositoryImpl is injected at
          com.yangbong.damedame.di.RepositoryModule.bindsSelectCharacterRepository(repository)
      com.yangbong.domain.repository.SelectCharacterRepository is injected at
          com.yangbong.set_profile.SelectCharacterViewModel(selectCharacterRepository)
      com.yangbong.set_profile.SelectCharacterViewModel is injected at
          com.yangbong.set_profile.SelectCharacterViewModel_HiltModules.BindsModule.binds(arg0)
      @dagger.hilt.android.internal.lifecycle.HiltViewModelMap java.util.Map<java.lang.String,javax.inject.Provider<androidx.lifecycle.ViewModel>> is requested at
          dagger.hilt.android.internal.lifecycle.HiltViewModelFactory.ViewModelFactoriesEntryPoint.getHiltViewModelMap() [com.yangbong.damedame.DameDameApplication_HiltComponents.SingletonC → com.yangbong.damedame.DameDameApplication_HiltComponents.ActivityRetainedC → com.yangbong.damedame.DameDameApplication_HiltComponents.ViewModelC]

---
구글링 결과
data module 과 app module 모두 의존성을 추가하면 된다고 하는데, 모두 의존성을 추가해준 것 같은데 제가 놓친 부분이 있었을까요 ?

**Comment by onseok** [comment]
`RemoteSelectCharacterDataSource`가 제대로 주입되지 않은 것 같습니다.

혹시 아직 에러가 해결되지 않았다면 원격 요청 부탁드립니다.
